iftk/helpers/microphone.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import asyncio
+
+import pyaudio
+
+
+async def microphone_async(
```

Comment:
1) Let's rename this file to `pyaudio.py`

2) Let's just call this `microphone`. Since our library is async by default in most cases, we can generally leave off `_async` suffixes, I think.